README.md

```diff
@@ -319,6 +320,23 @@ Committee has few options which enable convert request data.
 By default committee save converted data to `committee.params` and rails does not read it.
 So we need to save converted value to `'action_dispatch.request.request_parameters'` because rails create parameter from this value.
 
+### Streaming Response Validation
```

Comment:
If I recall correctly, I think using the BodyProxy would also result in higher memory usage (because the whole response will be held in memory). For individual requests this might be alright, but in aggregate it might really add up. We should probably at least mention this as something to be aware of. What do you think?

Reply:
Yes, we’re on the same page. It depends on the response body size, but it often eats a lot of memory.

> We should probably at least mention this as something to be aware of. What do you think?

That sounds helpful, and I agree with your suggestion.
On the other hand, should we consider supporting an option to discard the response (i.e., skip validation)?